aries_cloudagent/transport/inbound/http.py

```diff
@@ -15,8 +15,6 @@
 class HttpTransport(BaseInboundTransport):
     """Http Transport class."""
 
-    is_external = False
-
     def __init__(self, host: str, port: int, create_session, **kwargs) -> None:
```

Comment:
Should this follow the same flow as the `WsTransport` class and include `is_external` on the constructor?